pom.xml

```diff
@@ -21,28 +21,48 @@
 
   <!-- Change: groupId, artifactId, version -->
   <groupId>org.superbiz</groupId>
-  <artifactId>tomee-rest-arquillian</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <artifactId>tomee-jaxrs-starter-project</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
 
   <packaging>war</packaging>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>${arquillian.version}</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
-      <groupId>org.apache.openejb</groupId>
+      <groupId>org.apache.tomee</groupId>
       <artifactId>javaee-api</artifactId>
-      <version>${openejb.javaee.api}</version>
+      <version>${tomee.javaee.api}</version>
       <scope>provided</scope>
     </dependency>
+    <!-- NOTE geronimo-osgi-locator required by JAX-RS client from CXF unless JAX-RS client jar proceeds Java EE API jar -->
     <dependency>
-      <groupId>org.apache.openejb</groupId>
-      <artifactId>arquillian-tomee-embedded</artifactId>
-      <version>${tomee.version}</version>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-osgi-locator</artifactId>
+      <version>1.1</version>
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.apache.openejb</groupId>
-      <artifactId>tomee-jaxrs</artifactId>
-      <version>${tomee.version}</version>
+      <groupId>org.apache.cxf</groupId>
+      <artifactId>cxf-rt-rs-client</artifactId>
+      <version>${cxf.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <!-- NOTE JAX-RS JSON integration must be provided on client side -->
+    <dependency>
+      <groupId>org.apache.johnzon</groupId>
+      <artifactId>johnzon-jaxrs</artifactId>
```

Comment:
Again, only needed for the remote adapter. The reason for the very specific dependencies for that scenario was to limit number of items on the classpath. I figured out exactly which jars were needed.